jaraco/text/__init__.py

```diff
@@ -1,14 +1,14 @@
 import functools
 import itertools
 import re
+import sys
 import textwrap
-
 from typing import Iterable
 
-try:
-    from importlib.resources import files  # type: ignore
-except ImportError:  # pragma: nocover
-    from importlib_resources import files  # type: ignore
+if sys.version_info >= (3, 9):
+    from importlib.resources import files
+else:  # pragma: nocover
+    from importlib_resources import files
```

Comment:
Historically, I've been resistant to using this logic, because it feels like using a secondary signal. And typically, I'm averse to changing logic to accommodate types. In this case, I'm happy to follow the popular convention. Thanks!